ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/clientlibs/site/js/termsandconditionsview.js

```diff
@@ -62,10 +62,12 @@
                     threshold: [1],
                 })
                 function onIntersection ([{isIntersecting}]) {
+                    const isDisabled = self._model._jsonModel?.enabled && !self._model._jsonModel?.readOnly;
```

Comment:
The name of the variable 'isDisabled' seems to be a misleading. Shouldn't it be isEnabled?

Reply:
Implemented!